hs_swat_modelinstance/models.py

```diff
@@ -46,12 +46,18 @@ def get_swat_model_objectives(self):
 
     @classmethod
     def _add_swat_objective(cls, model_objective, objectives):
```

Comment:
@mohamedmorsy 

> After fixing the versioning problem, now these integer values pass but ignored in the validation method

I not sure what you mean by 'ignored in the validation method'. Can you explain how it is ignored?